waku/waku_test.go

```diff
@@ -1638,7 +1696,9 @@ func TestMailserverCompletionEvent(t *testing.T) {
 
 	w2 := New(nil, nil)
 	require.NoError(t, w2.Start(nil))
-	defer w2.Stop() // nolint: errcheck
+	defer func() {
+		handleError(t, w2.Stop())
+	}()
```

Comment:
[Error handling is good](https://github.com/status-im/status-go/pull/1957/files#r417328424)